### PR TITLE
#302: Fix the addresses

### PR DESCRIPTION
**Pull request by hani-ibrahim**
working on #302 
The shipping and billing address were flipped when updating the checkout request

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutSummaryViewController.swift [hunk 1]
```diff
@@ -162,8 +162,8 @@ extension CheckoutSummaryViewController {
 
     func createCheckout(cartId: String, completion: CheckoutCompletion) {
         checkout.client.createCheckout(cartId,
-            billingAddressId: checkoutViewModel.selectedShippingAddress?.id,
-            shippingAddressId: checkoutViewModel.selectedBillingAddress?.id) { checkout in
+            billingAddressId: checkoutViewModel.selectedBillingAddress?.id,
+            shippingAddressId: checkoutViewModel.selectedShippingAddress?.id) { checkout in
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryViewController.swift]: I think I fixed something like this before. I still see some differences, maybe you could look at this?

```
~/Documents/atlas-ios/AtlasUI issue-302 $ ags 'billingaddress' -c
AtlasUI/AtlasCheckout.swift:2
AtlasUI/CheckoutSummaryActionsHandler.swift:10
AtlasUI/CheckoutSummaryMainStackView.swift:9
AtlasUI/CheckoutSummaryViewController.swift:6
AtlasUI/CheckoutViewModel.swift:7

~/Documents/atlas-ios/AtlasUI issue-302 $ags 'shippingaddress' -c
AtlasUI/AtlasCheckout.swift:2
AtlasUI/CheckoutSummaryActionsHandler.swift:8
AtlasUI/CheckoutSummaryMainStackView.swift:10
AtlasUI/CheckoutSummaryViewController.swift:6
AtlasUI/CheckoutViewModel.swift:7
```

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/CheckoutSummaryViewController.swift]: yeah there was a problem in another place.. fixed now

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**